net-misc/icaclient/icaclient-22.11.0.19.ebuild

```diff
@@ -0,0 +1,260 @@
+SRC_URI="amd64? ( linuxx64-${PV}.tar.gz )
+	x86? ( linuxx86-${PV}.tar.gz )"
+
+LICENSE="icaclient"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="l10n_de l10n_es l10n_fr l10n_ja l10n_zh-CN"
+RESTRICT="mirror strip fetch"
+
+ICAROOT="/opt/Citrix/ICAClient"
+
+QA_PREBUILT="${ICAROOT#/}/*"
+
+# we have binaries for two conflicting kerberos implementations
+# https://bugs.gentoo.org/792090
+# https://bugs.gentoo.org/775995
+REQUIRES_EXCLUDE="
+	libgssapi.so.3
+	libgssapi_krb5.so.2 libkrb5.so.3
+"
+# we have binaries which wouls still support gstreamer:0.10
+REQUIRES_EXCLUDE="${REQUIRES_EXCLUDE}
+	libgstapp-0.10.so.0
+	libgstbase-0.10.so.0
+	libgstinterfaces-0.10.so.0
+	libgstpbutils-0.10.so.0
+	libgstreamer-0.10.so.0
+"
+RDEPEND="
+	|| (
+		>=app-accessibility/at-spi2-core-2.46.0:2
+		dev-libs/atk
+	)
+	app-crypt/libsecret
+	dev-libs/glib:2
+	dev-libs/libxml2
+	media-fonts/font-adobe-100dpi
+	media-fonts/font-misc-misc
+	media-fonts/font-cursor-misc
+	media-fonts/font-xfree86-type1
+	media-fonts/font-misc-ethiopic
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/gst-plugins-base:1.0
+	media-libs/gstreamer:1.0
+	media-libs/libogg
+	media-libs/libpulse
+	media-libs/libvorbis
+	media-libs/mesa
+	media-libs/speex
+	net-libs/libsoup:2.4
+	net-libs/webkit-gtk:4
+	sys-apps/util-linux
+	sys-libs/libcxx
+	sys-libs/libcxxabi
+	sys-libs/llvm-libunwind
+	sys-libs/zlib
+	virtual/krb5
+	virtual/libudev
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/gtk+:3
+	x11-libs/libX11
+	x11-libs/libXaw
+	x11-libs/libXcomposite
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXinerama
+	x11-libs/libXmu
+	x11-libs/libXrender
+	x11-libs/libXt
+	x11-libs/pango
+"
+DEPEND=""
+
+pkg_nofetch() {
+	elog "Download the client file ${A} from
+	https://www.citrix.com/downloads/workspace-app/"
+	elog "and place it into your DISTDIR directory."
+}
+
+pkg_setup() {
+	case ${ARCH} in
+		amd64)
+			ICAARCH=linuxx64
+		;;
+		x86)
+			ICAARCH=linuxx86
+		;;
+		*)
+			eerror "Given architecture is not supported by Citrix."
+		;;
+	esac
+}
+
+src_unpack() {
+	default
+	mv "${WORKDIR}/${ICAARCH}/${ICAARCH}.cor" "${S}" || die
+}
+
+src_prepare() {
+	default
+	rm lib/UIDialogLibWebKit.so || die
+
+	# We need to avoid module.ini file getting added to the package's
+	# content because media-plugins/hdx-realtime-media-engine modifies
+	# this file on installation. See pkg_postinst()
+	mv nls/en/module.ini "${T}" || die
+}
+
+src_install() {
+	local bin tmpl dest
+
+	dodir "${ICAROOT}"
+
+	exeinto "${ICAROOT}"
+	doexe *.DLL libproxy.so wfica AuthManagerDaemon PrimaryAuthManager selfservice ServiceRecord
+
+	exeinto "${ICAROOT}"/lib
+	doexe lib/*.so
+
+	for dest in "${ICAROOT}"{,/nls/en{,.UTF-8}} ; do
+		insinto "${dest}"
+		doins nls/en.UTF-8/eula.txt
+	done
+
+	insinto "${ICAROOT}"
+	doins -r usb
+
+	insinto "${ICAROOT}"/config
+	# nls/en/*.ini is being handled by pkg_postinst()
+	doins config/* config/.*
```

Comment:
```
>>> Install net-misc/icaclient-22.11.0.19 into /var/tmp/portage/net-misc/icaclient-22.11.0.19/image
Traceback (most recent call last):
  File "/usr/lib/portage/python3.11/doins.py", line 609, in <module>
    sys.exit(main(sys.argv[1:]))
             ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 598, in main
    if _doins(
       ^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 441, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 375, in install_file
    return self._ins_runner.run(source, dest_dir)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 183, in run
    sstat = os.stat(source)
            ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: b'config/.*'
 * ERROR: net-misc/icaclient-22.11.0.19::gentoo failed (install phase):
 *   doins failed
 * 
```

Reply:
Ok that is a wildcard which does not yield anything. I did not try with python 3.11 and maybe some new portage, maybe this creates that nice exception. An exception that should maybe rather be caught in portage and turned into a warning?

Reply:
I will remove the thing from this bump, but likely the other ones would be affected as well. Before fixing them we could likely use a bug to discuss whether we should improve portage here. It somehow changed, maybe with python changing.
@juippis maybe you can create such a bug mention the python and portage version, and maybe create a little example ebuild with an empty wildcard

Reply:
i included another commit to hopefully deal with that, switching all versions to not trying to install `config/.*`

that is a noop since the archives we get all do not contain files matching that wildcard, resolving thread ... not sure we want to raise this with portage itself